CMSgov/easi-app
# [EASI-3204] Feature/easi 3204 migrate contract number

# EASI-3204, EASI-3967, EASI-4037, EASI-4050


## Changes and Description

- Migrate existing data from `contract_numbers` column in `system_intakes` table to new linking table
- Updates to graphql types to allow for retrieving/saving multiple contract numbers on a single system intake
- UI handling and displaying of multiple, comma-separated contract numbers



## How to test this change

- Navigate to IT Gov Requests
- Create new system intake
- Proceed to funding page, and at the bottom select one of the options that displays the fields for entering contractor/contract number/contract date
- Try to add multiple, comma-separated contracts
- Save (or Next)
- Go back to funding page and confirm contract numbers are still there
- ** Note: the contract numbers may not be in the same order as they were entered - this is something to revisit (@ClayBenson94 @mynar7)



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
